themes/brainy/brainy.theme.bash

```diff
@@ -136,7 +136,7 @@ ___brainy_prompt_ruby() {
 
 ___brainy_prompt_todo() {
 	[ "${THEME_SHOW_TODO}" != "true" ] \
-		|| [ -z "$(which todo.sh)" ] && return
+		|| [ -z "$(command -v todo.sh)" ] && return
```

Comment:
same as the `atomic` theme (actually there is so much code dup here, we should split the logic to a separate file, but for now its enough)